### PR TITLE
[dash]: Add TAG for ACL

### PR DESCRIPTION
**Pull request by Pterosaur**

### CODE REVIEW

**Review by kcudnik — APPROVED** [review]

**Comment by oleksandrivantsiv** [comment]
@marian-pritsak can you please review this PR?

File: experimental/saiexperimentaldashacl.h [hunk 1]
```diff
@@ -110,6 +110,38 @@ typedef enum _sai_dash_acl_rule_attr_t
      */
     SAI_DASH_ACL_RULE_ATTR_DASH_ACL_GROUP_ID,
 
+    /**
+     * @brief Ternary matched key dst_tag
+     *
+     * @type sai_uint32_t
```
Comment (marian-pritsak) [inline experimental/saiexperimentaldashacl.h:116]: DASH behavioral model needs to be changed to make this attribute's type sai_object_list_t, per discussion with @prsunny

File: experimental/saiexperimentaldashtag.h [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+{
+    /**
+     * @brief Start of attributes
+     */
+    SAI_DST_TAG_ENTRY_ATTR_START,
+
+    /**
+     * @brief Action set_dst_tag parameter TAG_MAP
+     *
+     * @type sai_uint32_t
```
Comment (marian-pritsak) [inline experimental/saiexperimentaldashtag.h:68]: DASH behavioral model needs to be changed to make this attribute's type sai_object_list_t, per discussion with @prsunny

**Review by marian-pritsak — CHANGES_REQUESTED** [review]

File: experimental/saiexperimentaldashacl.h (hunk 1 above)
Comment (Pterosaur) [reply experimental/saiexperimentaldashacl.h:116]: As you know, this SAI header is automatically generated from P4. So, if we discard the bitmap design, should we also leverage a customized MACRO like `#define LIST_MATCH list`?

**Comment by Pterosaur** [comment]
The ACL Tag will be re-designed.

**Pull request closed without merging**